### ISSUE TITLE
MPII evaluation

### ISSUE THREAD

**Issue by syusukee** [open-mmlab/mmpose#145]
hello, I've read the evaluation code for MPII dataset, it calculated about the average accracy for all the keypoints. However, on its official website, the evaluation has been divided into two parts: single-person and multi-person, a little bit different from this repo. So does this repo use the same evaluaion tools as the official realization or only calculate the averaged results for all keypoints in all images? Hope for your reply! Thanks a lot!!

**Comment by jin-s13** [comment]
Yes. This repo uses the standard evaluation metrics as the official codes (single-person pose estimation). The training set contains 22246 people & 14679 images, the validation set contains 2958 people & 2729 images. The train/valid split follows Tompson, which is the standard setting.

**Comment by syusukee** [comment]
Thanks for your immediate reply! So if I want to get the multi-person evaluation, should I need to rewrite the evaluation code?

**Comment by jin-s13** [comment]
Yes, you may also need to process the original .mat file and select the `multi-person' part.

**Comment by syusukee** [comment]
I'll try it. Really appreciate your reply!!
